theme-release-110.rst

```diff
@@ -0,0 +1,91 @@
+.. post:: October 18, 2022
+    :tags: theme, release
+    :author: Benjamin
+    :location: Malmö, Sweden
+
+.. meta::
+    :description lang=en:
+        Information on sphinx-rtd-theme version 1.1.0
+
+
+Announcing sphinx_rtd_theme 1.1.0
+=================================
+
+We are happy to announce the release of new version of our theme, `sphinx-rtd-theme`_.
+In this release, we have focused on bug fixes, backwards compatibility, and making the way for future releases.
+
+.. _sphinx-rtd-theme: https://sphinx-rtd-theme.readthedocs.io/en/stable/
+
+
+Changes and new features 💄
+---------------------------
+
+Visually, we have a couple of small tweaks that most people won't notice unless we mention them here 😇
+The objective of 1.1 release is to maintain backwards compatibility, and that also goes for the visual parts.
+
+* The ``<kbd>`` tag now has a nicer styling.
+* Breadcrumb styling has been updated.
+* Table cells with multiple paragraphs now have a correct formatting.
+* Definition lists rendered correctly in API docs.
+* Issues with citation rendering and styling fixed.
+* Long URLs break into several lines instead of overflowing.
+
+.. image:: img/sphinx-rtd-theme-110-screenshot1.png
+   :scale: 12%
+   :align: right
+
+.. image:: img/sphinx-rtd-theme-110-screenshot2.png
+   :scale: 16%
+   :align: right
+
+.. image:: img/sphinx-rtd-theme-110-screenshot3.png
+   :scale: 16%
+   :align: right
+
+In the engine room, we have ensured the long-term stability for users of the theme by putting upper bounds on ``Sphinx<6`` and ``docutils<0.18``.
+
+We also fixed an issue that caused the theme to fail when Sphinx ``5.2.0.post0`` was released and will ensure that this doesn't happen again.
+
+`Read the full changelog <changelog>`_
+
+
+.. _changelog: https://sphinx-rtd-theme.readthedocs.io/en/stable/changelog.html
```

Comment:
I don't think this is valid rST syntax. These links don't work.

Reply:
Thanks! This came into my muscle memory a long time ago, and I'm still try to get rid of it. I wish it worked :upside_down_face: